test/thermo/BinarySolutionTabulatedThermo_Test.cpp

```diff
@@ -52,6 +52,9 @@ TEST_F(BinarySolutionTabulatedThermo_Test,interp_h)
     {
         set_defect_X(xmin + i*dx);
         EXPECT_NEAR(expected_result[i], test_phase->enthalpy_mole(), 1.e-6);
+        // enthalpy is temperature-independent
```

Comment:
This is correct for the cti file on which this test is built, but am I correct that it is not necessarily true of the phase model (i.e. the excess terms are added to the species "standard state" terms, which are calculated according to whatever is in the cti file?) 

If I'm correct about that, there are two rather minor concerns:
1. The test could easily be broken if the underlying cti file is updated w/ temperature-dependent species thermo.  Thinking while I write, here, but this is actually a good thing, in that the cti probably shouldn't be changed in that way.  It is good to be able to verify that temperature updates and composition updates are 'disentangled,' and the temperature-independent thermo provides a good pathway for this.
2. Also a very minor concern, but someone reading this test suite to learn more about the class might get the wrong impression.  Chances are vanishingly small that this would happen, particularly as there are other, better places to learn about the class.  But it crossed my mind...

Reply:
Correct, that is a property of the example input file which sets the standard state to be h(T) = 0, not of the model as a whole. In general, I would expect changes to the test data files to cause some of the tests to fail -- if you can change the data files and all the tests still succeed, I'd argue that the tests aren't checking the right things.

I will update the comment to indicate *why* the enthalpy is temperature independent in this particular case, to avoid any confusion.